### ISSUE TITLE
On applying Resize(Larger Text) "Organize a Friends Get together" list option in task section is getting truncated: A11y_.NET 9 template_Home_Resize

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#27423]
### GitHub Tags:
#A11yTCS; #A11yMAS; #WCAG1.4.4; #A11ySev2; #iOS; #SH_.NET9template_iOS_Jan2025; #.NET Core; #Device; #Resize; #AINotInscope; #iphone;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
iOS Version: 18.2

### Prerequisites:
Settings -> Accessibility ->Display and text size -> Larger text ->Text size need to be increased to maximum level.

### Repro Steps:

1. Go to setting and apply the above mentioned prerequisites,
2. Install and open ".NET 9 template" app.
3. "Home Screen" will open.
4. Swipe till "Task section"
5. Observe that on applying Resize (Larger Text) "Organize a Friends Get together" list option in task section is getting truncated.

### Actual Result:
On applying Resize (Larger Text) "Organize a Friends Get together" list option in task section is getting truncated.

### Expected Result:
On applying Resize (Larger Text) "Organize a Friends Get together" list option in task section should not truncate and should be clearly visible.

### User Impact:
Low vision users who use Resize resolution will find it difficult to read the information as it is getting truncated.

### Attachment:

https://github.com/user-attachments/assets/680fe971-9c7f-4bde-a6d6-43af4ce284cc

**Comment by PoojaNamde** [comment]
@PureWeen  Could you please confirm if the "Organize a Friends Get Together" list option in the task section has been removed? We need to verify this so that we can close the bug, as we are unable to see that option on the screen.

**Comment by tj-devel709** [comment]
@PoojaNamde Hi, I'm having trouble uploading the images but they are still showing up for me and resizing accordingly when the size is increased.

**Comment by merriemcgaw** [comment]
What is the latest on this issue @tj-devel709 @PureWeen ?

**Comment by tj-devel709** [comment]
Was asked to close this one on behalf of @PoojaNamde
